portfolio/src/main/webapp/script.js

```diff
@@ -42,14 +42,14 @@ $('.carousel').carousel({
 
 function getMessagesFromServer() {
   fetch('/data').then(response => response.json()).then((messages) => {
+    console.log(messages);
```

Comment:
Remove this line, there shouldn't be any console.log (s) in the final version.

Reply:
Solved